src/headers.rs

```diff
@@ -14,6 +14,7 @@ use hyper::header::{Header, HeaderFormat, parsing};
 ///
 /// This is an internal type not meant for general public usage, as its implementation limited in
```

Comment:
I've already upstreamed a better version of the ContentDisposition header, but clearly I never updated formdata to use it.  I'll get onto that.